figures/S5/scripts/S5.r

```diff
@@ -0,0 +1,293 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2))) # plotting
+suppressPackageStartupMessages(suppressWarnings(library(dplyr))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(patchwork))) # figure composition
+suppressPackageStartupMessages(suppressWarnings(library(cowplot))) # figure composition
+suppressPackageStartupMessages(suppressWarnings(library(RcppTOML))) # parsing config file
+suppressPackageStartupMessages(suppressWarnings(library(lattice))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(RColorBrewer))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(ComplexHeatmap))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(ggplotify))) # grob
+suppressPackageStartupMessages(suppressWarnings(library(viridis))) # color
+suppressPackageStartupMessages(suppressWarnings(library(platetools))) # make plate plot
+suppressPackageStartupMessages(suppressWarnings(library(circlize)))
+suppressPackageStartupMessages(suppressWarnings(library(reshape2))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(stringr))) # string manipulation
+suppressPackageStartupMessages(suppressWarnings(library(purrr))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(VennDiagram))) # venn diagram
+suppressPackageStartupMessages(suppressWarnings(library(tidyverse))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(ggvenn))) # venn diagram
+suppressPackageStartupMessages(suppressWarnings(library(grid))) # grid
+suppressPackageStartupMessages(suppressWarnings(library(Polychrome)))
```

Comment:
Consider adding a comment here like with the rest of the imports

Reply:
addressed